api/tests/certification/shared/integration/infrastructure/repositories/issue-report-categories-repository_test.js

```diff
@@ -1,18 +1,43 @@
-import { expect, databaseBuilder } from '../../../../../test-helper.js';
+import { expect, databaseBuilder, catchErr } from '../../../../../test-helper.js';
 import * as issueReportCategoryRepository from '../../../../../../src/certification/shared/infrastructure/repositories/issue-report-category-repository.js';
+import { CertificationIssueReportCategory } from '../../../../../../lib/domain/read-models/CertificationIssueReportcategory.js';
+import { NotFoundError } from '../../../../../../lib/domain/errors.js';
 
 describe('Integration | Repository | Issue Report Categories', function () {
   describe('#get', function () {
-    it('should return a certification issue report', async function () {
-      // given
-      const issueReportCategory = databaseBuilder.factory.buildIssueReportCategory({ category: 'OTHER' });
-      await databaseBuilder.commit();
+    describe('when the issue report category name exists in the database', function () {
```

Comment:
Pas forcé de specifier 'database'
```suggestion
    describe('when the issue report category name exists', function () {
```